pkgs/top-level/all-packages.nix

```diff
@@ -5032,6 +5032,8 @@ with pkgs;
 
   qtikz = libsForQt5.callPackage ../applications/graphics/ktikz { };
 
+  quantomatic = callPackage ../applications/science/physics/quantomatic { };
```

Comment:
This should probably be further down in the `### APPLICATIONS` section, between `qtscrobbler` and `quassel`